Update api_docs for dynamic_embedding

Lifann opened the pull request:
Update api_docs for dynamic_embedding

rhdong commented:
it's duplicated for ：
`tfra.dynamic_embedding.safe_embedding_lookup_sparse`
`tfra.dynamic_embedding.python.ops.safe_embedding_lookup_sparse`

Lifann commented:
> it's duplicated for ：
> `tfra.dynamic_embedding.safe_embedding_lookup_sparse`
> `tfra.dynamic_embedding.python.ops.safe_embedding_lookup_sparse`

I've removed the duplicated APIs, and dropped the `dynamic_embedding.python.xxx`.
